src/Umbraco.Web.UI.Client/src/views/propertyeditors/grid/grid.html

```diff
@@ -1,6 +1,7 @@
 <div ng-controller="Umbraco.PropertyEditors.GridController" class="umb-grid umb-editor clearfix" id="umb-grid">
 
     <umb-editor-sub-header>
+        <label style="float: right;"><input type="checkbox" ng-model="showNav" />Nav</label>
```

Comment:
So sorry again. The sub-header-content-right isolates scope, so can't use it.
Float right doesn't work for some reason. CSS love please. ;)